app/models/post.rb

```diff
@@ -4,6 +4,11 @@ class Post < ApplicationRecord
   has_many :likes
   validates :title, :text, presence: true
 
+  validates :title, presence: { message: 'Title must not be blank' },
```

Comment:
- [ ] It would be better if we ensure that the field name (title) should be present and not empty. And the best way to do that is `presence: true` instead of `presence: { message: 'Title must not be blank' }` as we have currently